{
  "Header": {
    "Original": "CVE-2020-25575 (An issue was discovered in the failure crate through 0.1.5 for Rust. I ...)",
    "Line": 266972,
    "ID": "CVE-2020-25575",
    "Description": "(An issue was discovered in the failure crate through 0.1.5 for Rust. I ...)"
  },
  "Annotations": [
    {
      "Original": "- rust-failure \u003cunfixed\u003e (unimportant; bug #969839)",
      "Line": 266973,
      "Type": "package",
      "Package": "rust-failure",
      "Kind": "unfixed",
      "Severity": "unimportant",
      "BugNo": 969839
    },
    {
      "Original": "NOTE: https://rustsec.org/advisories/RUSTSEC-2020-0036.html",
      "Line": 266974,
      "Type": "NOTE",
      "Description": "https://rustsec.org/advisories/RUSTSEC-2020-0036.html"
    },
    {
      "Original": "NOTE: https://github.com/rust-lang-nursery/failure/issues/336",
      "Line": 266975,
      "Type": "NOTE",
      "Description": "https://github.com/rust-lang-nursery/failure/issues/336"
    },
    {
      "Original": "NOTE: This CVE ID is merely for the fact that the crate is unmaintained",
      "Line": 266976,
      "Type": "NOTE",
      "Description": "This CVE ID is merely for the fact that the crate is unmaintained"
    }
  ]
}